provision/internal/operator/terraform/tfcmd.go

```diff
@@ -50,7 +51,10 @@ func initArgs(p types.ProviderType, cfg map[string]interface{}, clusterDir strin
 		// TODO remove this condition when fully migrated to modules
 		if m := tfMod(p); m != "" {
 			args = append(args, fmt.Sprintf("-from-module=%s", m))
-		}
+		}}
```

Comment:
Please run `gofmt -s -w` no this file, otherwise the CI/CD validation will fail.